nvflare/app_common/executors/multi_process_executor.py

```diff
@@ -33,6 +33,7 @@
 from nvflare.fuel.utils.class_utils import ModuleScanner
 from nvflare.fuel.utils.component_builder import ComponentBuilder
 from nvflare.fuel.utils.network_utils import get_open_ports
+from nvflare.private.fed.simulator.simulator_app_runner import SimulatorClientRunManager
```

Comment:
I think code from app_common can't depend on private?
We could set field in fl_ctx to avoid this import.

Reply:
changed the way to determine simulator mode. removed this import.